### DISCUSSION TITLE
Need Help getting full part of Mod to work.

### DISCUSSION THREAD

**Discussion by Moooooooonnnn** [CosmicHikari/Weather2-Remastered#111] [Questions] [answered]
I seem to be having trouble getting the full part of the Mod to work. I am able to play it, but I seem to be missing some items, such as: (different Radar tiers, graphics, etc). I was hoping to get some help as to what I am doing wrong. I have deleted my entire mods folder, and the Weather2 config folder multiple times, and reinstalled 1.12.2 forge. If anyone could help me, id be really grateful.

**Comment by Mrbt0907** [comment, marked as answer]
That version of the mod is found in my discord server in the current indev.

They do not exist in the current alpha.

**Reply by Moooooooonnnn** [reply]
Ah ok, thank you. Do you mind if I join whenever I can?

**Reply by Mrbt0907** [reply]
Anytime. https://discord.gg/trNQFjZ26w
